server/src/controllers/menuControllers.js

```diff
@@ -8,6 +8,32 @@ class MenuControllers {
       data: menu
     });
   }
+
+  // Post Menu for the single day
+  static postMenu(req, res) {
+    if (!req.body.name) {
+      return res.status(400).send({
+        status: res.statusCode,
+        message: "Menu name is required"
+      });
+    }
+    const newMenu = {
+      id: req.body.id,
```

Comment:
# Critical
Users should never supply `id`. I think I suggested using `shortid` or `uuid` earlier.